src/replication/DealReplicationWorker.ts

```diff
@@ -316,7 +354,7 @@ export default class DealReplicationWorker extends BaseService {
               provider: provider,
               state: {
                 $in: [
-                  'proposed', 'published'
+                  'proposed', 'published', 'active'
```

Comment:
why is active considered pending deals?

Reply:
This could happen when the deal making process takes a very long time (for example very large number of deals to make, or daemon stopped for several hours or longer), some deals might already became active.